gnome-initial-setup/pages/location/gis-location-page.c

```diff
@@ -687,6 +687,25 @@ gis_location_page_constructed (GObject *object)
 
   update_timezone (page);
 
+  /*
+   * When running on a Live session, it's important to try to make the best
+   * usage of the system clock. Timedate1 has a handy method for that.
+   */
+  if (gis_driver_is_live_session (GIS_PAGE (object)->driver)) {
+    timedate1_call_set_local_rtc_sync (priv->dtm,
+                                       TRUE,
+                                       TRUE,
+                                       FALSE,
+                                       NULL,
+                                       &error);
+
+    if (error) {
+      g_warning ("Error calling SetLocalRTC(): %s", error->message);
+      g_clear_error (&error);
+      exit (1);
```

Comment:
Let's not terminate the whole process if calling this method fails… It is not the end of the world if this doesn't work, and if this path is reached for some unexpected reason then our users will have totally unusable live USBs and will send us angry emails.